serverless/serverless-azure-functions
# ci: .NET Core Integration tests & Workflow fixes

## What did you implement:

Adds .NET core integration tests that will be triggered 2x daily as well as any pushes to branch with pattern `*/trigger-int-tests`



## How did you implement it:

- Added templates for .NET Core versions 3.1 & 2.2 for Linux and Windows
- Updated the workflow to create jobs for all multi-os runtimes, keeping Python in its own job



## How can we verify it:

Check out and push to a branch with pattern `*/trigger-int-tests` and go to `Actions` page on repo



## Todos:

_**Note: Run `npm run test:ci` to run all validation checks on proposed changes**_

       **Validate via `npm run lint`**  
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
       **Validate via `npm test`**

**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO
